support/test/integration/docker-compose.yml

```diff
@@ -38,7 +38,7 @@ services:
       dockerfile: support/Dockerfile
       cache_from:
         - augurproject/augur-core:latest
-    entrypoint: [ "npm", "run", "test:integration"]
+    entrypoint: [ "npx", "mocha", "output/tests-integration/**/*.js", "--no-timeouts", "--require", "source-map-support/register"]
```

Comment:
@pgebheim This change may interest you.  I had to make this change because I found that uncommenting the lines below no longer caused node to wait for a debugger to attach.  I'm not against changing them back to `npm run ...`, but we'll need to figure out a way to get debugging docker images back as that is incredibly useful when troubleshooting CI test failures.